backend/src/BeyondRGB.py

```diff
@@ -88,20 +89,28 @@ def main():
                        (int(args.white_row), int(args.white_col)),
                        targ2ttype[args.target])
 
+    # Initialize btrgb file
+    btrgb_file = Btrgb()
+    
     # Setup packet
-    packet = build_packet(args.images, target, args.outpath)
+    packet = build_packet(args.images, target, args.outpath, btrgb_file)
+
 
     # Begin pipeline
     processing_pipeline(packet)
 
+    # Write btrgb file
+    packet.btrgb.write_to_file("123", args.outpath)
```

Comment:
Is "123" the name of the file? This should probably be "output" or "beyondrgb_output" or something

Reply:
123 is the "id" of the btrgb file. The output is always BeyondRGB_id.btrgb

Reply:
I still feel like this should probably be something else, but I'm not sure what the point of a file id is